src/ResourceManager/Insights/Commands.Insights/LogProfiles/RemoveAzureRmLogProfileCommand.cs

```diff
@@ -39,8 +37,18 @@ public class RemoveAzureRmLogProfileCommand : ManagementCmdletBase
 
         protected override void ProcessRecordInternal()
         {
-            WriteWarning("The output of this cmdlet will change. The cmdlet will not return anything in future releases.");
-            this.InsightsManagementClient.LogProfiles.DeleteAsync(logProfileName: this.Name, cancellationToken: CancellationToken.None).Wait();
+            WriteWarning("The type of the output will change to follow the pattern from other similar operations.");
```

Comment:
@gucalder can you provide any examples of similar operations so users will know?